src/cljstyle/format/ns.clj

```diff
@@ -385,6 +390,13 @@
     [(n/spaces indent-size) docstr]))
 
 
+(defn- render-attr-map
+  "Render ns data provided as an attr-map."
+  [opts attr-map]
+  (when attr-map
+    [(n/spaces indent-size) (indent/reindent attr-map opts)]))
```

Comment:
This doesn't work the way you want because `reindent` here doesn't have the context on the rest of the ns form. Instead, you could manually format the map entries with a basic `(n/spaces (inc indent-size))` interposed. This wouldn't fix everything but I think the indentation logic might catch the rest.

Reply:
Thanks for the feedback @greglook! I just force pushed a new implementation that now passes the (expanded) test. Let me know if you would like any other changes.